src/services/identity/default.ts

```diff
@@ -0,0 +1,36 @@
+import { AbstractIdentity } from "./IIdentity.js";
```

Comment:
This file is not on develop, so it coming back is probably a bad merge.

Reply:
Moved DefaultIdentity class to `src/services/identity/IIdenity.ts` file. ✅